hazelcast/src/test/java/com/hazelcast/client/config/XmlYamlClientConfigBuilderEqualsTest.java

```diff
@@ -116,8 +115,10 @@ public void testFullClientFailoverConfig() throws Exception {
     }
 
     private String readResourceToString(String resource) throws IOException {
-        InputStream xmlInputStream = getClass().getClassLoader().getResourceAsStream(resource);
-        return new String(IOUtil.toByteArray(xmlInputStream));
+        try (InputStream xmlInputStream = getClass().getClassLoader().getResourceAsStream(resource)) {
+            assert xmlInputStream != null;
+            return new String(xmlInputStream.readAllBytes());
```

Comment:
We should provide `UTF-8` as a second parameter.

Reply:
rev : d482fbd433dee4857cee916d35f7925f9766f36e